examples/mysql-galera/image/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:trusty
+
+# add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
+RUN groupadd -r mysql && useradd -r -g mysql mysql
+
+ENV PERCONA_XTRADB_VERSION 5.6
+ENV MYSQL_VERSION 5.6
+ENV TERM linux
+
+RUN apt-get update 
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y perl --no-install-recommends && rm -rf /var/lib/apt/lists/*
```

Comment:
Can you batch this apt-get install command with the apt-get install below, or is there a reason they need to be separate.